utils/process/src/lib.rs

```diff
@@ -80,12 +100,21 @@ pub enum ExeUnitExitStatus {
 #[derive(Clone)]
 pub struct ProcessHandle {
     process: Arc<SharedChild>,
+    // Windows process will be killed once `job_object` is dropped
+    #[allow(unused)]
+    #[cfg(windows)]
+    job_object: JobObject,
 }
 
 impl ProcessHandle {
     pub fn new(command: &mut Command) -> Result<ProcessHandle> {
+        let process = Arc::new(SharedChild::spawn(command)?);
+        #[cfg(windows)]
+        let job_object = JobObject::try_new_current()?;
```

Comment:
I left `new_process_group` unimplemented for Windows. `process_group` is roughly equivalent of Windows JobObject, but `new_process_group` is implemented for `Command` and I needed to store `job_object` somewhere, I could not extend `Command`, so instead I just crate `job_object` while creating `ProcessHandle`.